docs/developer-guide/prerequisites.md

```diff
@@ -0,0 +1,4 @@
+# Technical prerequisites to contributing
+
+-   [Node.JS](https://nodejs.org/en/) 8+.
```

Comment:
The correct name is “Node.js.” Also, 8+ isn't a legit requirement. Personally, I'm using Node.js 6, and I'm going to switch to Node.js 8 only when it became LTS. I think the most correct version is 4 because it's a minimal version for stylelint itself.

Reply:
You're right.